packages/calling/calling/jest.config.js

```diff
@@ -29,11 +29,6 @@ module.exports = {
       functions: 30,
       statements: 40,
     },
-    'src/Hooks': {
```

Comment:
In the latest pull, this change shouldn't reflect here

Reply:
This change was not present in the latest pull. We have probably forgot to remove the jest coverage exception when we removed hooks logic.

Reply:
I removed it in my commits, not sure why that's not reflecting

Reply:
No issues.